probe/core/src/main/java/org/jboss/weld/probe/Resource.java

```diff
@@ -251,7 +251,7 @@ protected void handle(HttpMethod method, JsonDataProvider jsonDataProvider, Stri
 
     private final String[] parts;
 
-    private final Handler handler;
+    private final transient Handler handler;
```

Comment:
`Handler` instances do not hold any state so it might be better to `Handler implements Serializable` instead?